xdg-ninja.sh

```diff
@@ -168,19 +238,25 @@ check_file() {
     FILENAME="$2"
     MOVABLE="$3"
     HELP="$4"
+    JSON_FILE="$5"
 
     check_if_file_exists "$FILENAME"
 
     case $? in
 
     0)
+        [ "$SKIP_OK" = false ] && [ "$OUTPUT_STYLE" = "json" ] && cat "$JSON_FILE" && return
         log SUCS "$NAME" "$FILENAME" "$HELP"
         ;;
 
     1)
         if [ "$MOVABLE" = true ]; then
+            EXIT_STATUS=$(expr $EXIT_STATUS + 1)
+            [ "$OUTPUT_STYLE" = "json" ] && cat "$JSON_FILE" > "$XDG_RUNTIME_DIR"/xdg-ninja"$NAME".json && return
```

Comment:
You are doing it wrong. Many files of the same program may be addressed is one JSON file. JSON files are for *programs* not for *files*